modules/gapi/test/cpu/gapi_ocv_stateful_kernel_tests.cpp

```diff
@@ -8,6 +8,30 @@
 #include <opencv2/gapi/cpu/core.hpp>
 #include <opencv2/gapi/streaming/cap.hpp>
 
+#include <opencv2/core.hpp>
+#include <opencv2/core/cvstd.hpp>
+#include <opencv2/video.hpp>
+
+
+namespace opencv_test
+{
+    struct SubstractBackgroundStateParams
```

Comment:
`SubtractBackground` is uncommon name for the feature, usually it is `BgSub` or something like that.

Reply:
Renamed to `BackSub` to comply with used names for substractor in the test